chat-commands.js

```diff
@@ -2921,7 +2921,9 @@ exports.commands = {
 		if (!user.hasConsoleAccess(connection)) {
 			return this.errorReply("/bash - Access denied.");
 		}
-
+		if (!target) {
+			return this.sendReply("/bash (Console Access) needs arguments to carry out execution by running a child process. \nExample Execution: /bash ls");
```

Comment:
Definetly isn't standard to use ``\n`` here. I think maybe under the command, just add like:
``bashhelp: ["/bash [argument] - Carries out execution by running a child process. Requires console access."],``

Then, in the command, just do: ``if (!target) return this.parse('/help bash');``